pkg/eks/util/validators.ts

```diff
@@ -1,15 +1,16 @@
 import { set } from '@shell/utils/object';
-import { EKSConfig, EKSNodeGroup } from 'types';
+import { EKSConfig, EKSNodeGroup, NormanCluster } from 'types';
 
 export interface CruEKSContext {
   t: Function,
   config: EKSConfig,
   nodeGroups: EKSNodeGroup[],
+  normanCluster: NormanCluster
 }
 
 const clusterNameRequired = (ctx: CruEKSContext) => {
   return (): string | null => {
-    return !ctx.config.displayName ? ctx.t('validation.required', { key: ctx.t('nameNsDescription.name.label') }) : null;
+    return !ctx.normanCluster.name ? ctx.t('validation.required', { key: ctx.t('nameNsDescription.name.label') }) : null;
```

Comment:
validate the name field that the user types instead of the displayName, which may come from eks API